browser/resources/settings/brave_help_tips_page/brave_help_tips_page.js

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Also missing "Copyright (c) 2019 The Brave Authors. All rights reserved.".

Reply:
Done.